libass/ass_shaper.c

```diff
@@ -613,7 +619,7 @@ shape_harfbuzz_process_run(GlyphInfo *glyphs, hb_buffer_t *buf, int offset)
  * \param glyphs glyph clusters
  * \param len number of clusters
  */
-static void shape_harfbuzz(ASS_Shaper *shaper, GlyphInfo *glyphs, size_t len)
+static int shape_harfbuzz(ASS_Shaper *shaper, GlyphInfo *glyphs, size_t len)
```

Comment:
Wouldn’t it be more consistent to make this return `bool` (true on success, false on failure)? I assume you’re mirroring the return value of `ass_shaper_shape` here, so shall we change _that_ to `bool` too?